MineColab.ipynb

```diff
@@ -42,7 +42,7 @@
         "\n",
         "---\n",
         "\n",
-        "The script below will run your server. You'll have to create a server first to be able to use it - don't worry, the scripts below will do the majority of the work for you."
+        "The script below will run your server. You'll have to create a server first to be able to use it - don't worry, the scripts below will do the majority of the work for you. You might also want to change the default region to your region, check below."
```

Comment:
@thecoder-001 you might wanna modify this one because I'm not sure what I should put here to tell people to change the region.

Reply:
yup, will do